pkgs/by-name/ao/ao3downloader/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  python312Packages,
+  fetchFromGitHub,
+}:
+
+python312Packages.buildPythonApplication rec {
+  pname = "ao3downloader";
+  version = "2025.6.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "nianeyna";
+    repo = pname;
```

```suggestion
    repo = "ao3downloader";
```
Do not rely on repo names matching program names